Tetris Clone C++/TetrisPiece.cpp

```diff
@@ -195,45 +214,31 @@ bool TetrisPiece::rotateCounterClockwise() {
 }
 
 bool TetrisPiece::offsetFree(bool offsetGrid[4][4], int colOff, int rowOff, bool lock) {
```

Comment:
we should just remove lock from the argument list of this member function, remove lines that say `if (lock) this->lock();`